.travis.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
@stefanpenner - Should be deleted

Reply:
https://github.com/emberjs/ember-test-helpers/pull/1098